components/scream/scripts/tests/CTestConfig.cmake

```diff
@@ -0,0 +1 @@
+../../CTestConfig.cmake
```

Comment:
Not familiar with how ctest/cmake parse this file, but shouldn't this be something like `include(../../CTestConfig.cmake)`?

Reply:
I think GitHub is just telling you this is a symlink.

Reply:
Ah you're right. It even said it when I clicked on "View File", but I missed it.